odl/solvers/nonsmooth/proximal_operators.py

```diff
@@ -270,8 +281,17 @@ def arg_scaling_prox_factory(sigma):
             The proximal operator of ``sigma * F( . * a)`` where ``sigma`` is
             the step size
         """
-        prox = prox_factory(sigma * scaling ** 2)
-        return (1 / scaling) * prox * scaling
+        scaling_square = scaling * scaling
+#       TODO Catch division by zero errors. The following doesn't work:
```

Comment:
Why not add some sort of check for that? It would at least improve the error message!

Reply:
As long as space elements are allowed as inputs, I am against doing this kind of (potentially expensive) checks unconditionally.